frontend/src/layouts/cameras/index.tsx

```diff
@@ -86,17 +189,51 @@ const DevicesContainer = () => {
         >
             {devices
                 .sort((a: Device, b: Device) => {
-                    const regex = /(\/dev\/video)(\d+)/;
-                    let pathA = a.cameras[0].path;
-                    let pathB = b.cameras[0].path;
+                    // Sort by the last number in bus info
+                    // TODO: maybe sort by device type too
```

Comment:
TODO?

Reply:
This is still alpha software 🤷

Reply:
Plenty of software still does have TODOs, so I am personally ok with that. However, I will work on this change anyway since it is super easy.

Reply:
Added.

```js
if (a.device_type === b.device_type)
     return (
          Number(regex.exec(pathA)![0]) -
          Number(regex.exec(pathB)![0]) ||
          hash(pathA) - hash(pathB)
     );
else return hash(a.device_type) - hash(b.device_type);
```

Reply:
Will eventually change to a slightly better approach, but this works great for now